ppafm/ocl/field.py

```diff
@@ -277,6 +277,70 @@ def to_file(self, file_path, clamp=None):
             io.save_vec_field(file_head, array[:, :, :, :3], self.lvec, data_format=ext)
             io.save_scal_field(file_head + "_w", array[:, :, :, 3], self.lvec, data_format=ext)
 
+    def _prepare_same_size_output_grid(self, array_in, in_place):
+        if in_place:
+            grid_out = self
```

Comment:
array_out = array_in ? Isn't it missing.

Reply:
It was redundant. The array pointer is stored inside the `grid_out` object.

Reply:
Sorry, for my ignorance, what does `in_place` mean and why we need to set `array_out` when `in_place` is False?
Good to go, after this.